serveradmin/servershell/static/js/servershell.js

```diff
@@ -252,4 +252,24 @@ servershell.get_object = function(object_id) {
 servershell.get_attribute = function(attribute_id) {
     return servershell.attributes.find(
         attribute => attribute.attribute_id === attribute_id);
-};
\ No newline at end of file
+};
+
+/**
+ * Save the status of attributes list
+ */
+servershell.save_hidden_attrlist = function(element) {
+    localStorage.setItem('attrlist_hidden', (element.attributes["aria-expanded"]["nodeValue"] === 'true'));
+};
+
+/**
+ * Check the local storage and toggle attributes list
+ *
+ * If it's not defined or set to not hidden, it triggers collapse function
+ * to show. Otherwise it keeps it closed.
+ */
+servershell.set_attrlist_state = function() {
```

Comment:
Why you chose `set_attrlist_state` is clear to me, however the `set` adds the connotation that you are able to set the state and not that it will automatically set.
Maybe `update_attrlist_visibility` or `sync_attrlist_visibility` might be a better choice of naming.

Reply:
Doing sync_attrlist_visibility